src/NeoSharp.Core/Blockchain/Processors/BlockProcessor.cs

```diff
@@ -109,5 +89,21 @@ public void Dispose()
             await _repository.AddBlockHeader(block.GetBlockHeader());
             await _repository.SetTotalBlockHeight(block.Index);
         }
+
+        private async Task<bool> ContainsBlock(UInt256 blockHash)
```

Comment:
Why is it private now?

Reply:
because it's not been used anywhere else. If need outside (that I don't believe) we can turn public again